src/shared/schema.ts

```diff
@@ -36,7 +36,7 @@ const extendedJoi: ExtendedJoi = Joi.extend((joi) => ({
   }
 }))
 
-const passwordRule = Joi.string().min(6).max(128).required()
```

Comment:
I think the password rule should remain at minimum 6 characters, for security™ 😅

Reply:
Yes, make sense to default to 6 chars. But should maybe be an env var setting.

Reply:
https://github.com/nhost/hasura-backend-plus/issues/132